sem/dashboard/static/dashboard.js

```diff
@@ -0,0 +1,300 @@
+                    indexAxis: 'x',
+                    pointRadius: 5,
+                    pointHoverRadius: 8,
+                    data: result.plot
+                }]
+            };
+
+            var option = {
+                parsing: false,
+                normalized: true,
+                animation: false,
+
+                scales: {
+                    yAxes: [{
+                        stacked: true,
+                        gridLines: {
+                            display: true,
+                            color: "rgba(255,99,132,0.2)"
+                        }
+                    }],
+                    xAxes: [{
+                        type: 'time',
+                        gridLines: {
+                            display: true,
+                            color: "rgba(255,99,132,0.2)"
+                        }
+                    }]
+                },
+                // elements: {
+                //     line: {
+                //         tension: 0,
+                //     }
+                // },
+                plugins: {
+                    decimation: {
+                        enabled: true,
+                        algorithm: 'lttb'
+                    },
+                    tooltip: {
+                        callbacks: {
+                            label: function(tooltipItem) {
+                                // console.log('hi');
+                                // console.log(tooltipItem);
+                                // console.log(result.data);
+                                // console.log(result.data[0]);
+                                var current_log = result.data[tooltipItem.dataIndex];
+                                var timestamp = current_log.time;
+                                var context = current_log.context;
+                                var extended_context = current_log.extended_context;
+                                var component = current_log.component;
+                                var func = current_log.function;
+                                var args = current_log.arguments;
+                                var severity_class = current_log.severity_class;
+                                var mssg = current_log.message;
+                                if (extended_context != null)
+                                {
+                                    return ["Extended Context: " + extended_context, "Component: " + component,  "Function: " + func, "Arguments: " + args, "Severity Class: " + severity_class, "Message: " + mssg];
+                                }
+                                else
+                                {
+                                    return ["Component: " + component,  "Function: " + func, "Arguments: " + args, "Severity Class: " + severity_class, "Message: " + mssg];
+                                }
+                            }
+                        },
+                    },
+                    zoom: {
```

Comment:
is there a way to make the selected area clearer? it's almost invisible in the monitor i'm using